src/ethereum-canister/tests/canister.rs

```diff
@@ -0,0 +1,53 @@
+use candid::Nat;
+
+mod test_canister;
+
+use crate::test_canister::{call, TestCanister};
+
+#[test]
+fn get_block_number() {
+    let canister = TestCanister::setup_ethereum_canister();
+
+    let block_num: (Nat,) = call!(canister, "get_block_number", ()).unwrap();
```

Comment:
Add `($canister:expr, $method:expr)` in `call!` and change this to `call!(canister, "get_block_number")`

Reply:
done